makefile.shade

```diff
@@ -296,13 +296,19 @@ var buildTarget = "compile"
         }
     }
 
-#cache-coherence
+#use-local-coherence description='Sets up environment variables to use Coherence at COHERENCE_PATH'
     @{
         if (string.IsNullOrEmpty(coherencePath))
         {
             throw new Exception("COHERENCE_PATH not specified.");
         }
 
+        universeCommitsFile = Path.Combine(coherencePath, UniverseCommitsFileName);
+        if (!File.Exists(universeCommitsFile))
+        {
+            throw new Exception("universeCommitsFile could not be found at " + UNIVERSE_COMMITS_FILE);
```

Comment:
`+ universeCommitsFile`

Change the beginning of the sentence to "Universe commits file".